data/human/human missions.txt

```diff
@@ -7455,3 +7455,573 @@ mission "Adelita 5"
+
+mission "Syndicate Business 3"
+	name "Visit Howard on <planet>"
+	description "Howard, from Megaparsec Inc., wants to tell you about an investment opportunity."
+	source
+		government "Syndicate"
+		attributes "core"
+		not planet "Hephaestus"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 2: done"
+		random < 40
+		credits > 40000000
+	on offer
+		conversation
+			`After an uneventful trip to the spaceport you return to the <ship> and see a message from Howard, the Megaparsec salesman who sold you a one million credit bond. The message says:`
+			`	"<first> <last>! My bank contact tells me you've been doing quite well for yourself, and I've come across an even bigger investment opportunity than last time! Meet me at my new office on <planet> (note the address, I've moved up since my promotion). Looking forward to chat!"`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+			choice
+				`	(Message him back that you're on the way.)`
+				`	(Send a message congratulating him on the promotion.)`
+				`	(Ignore and delete the message.)`
+					decline
+			`	You immediately receive an automated response:`
+			`	"Dear Valued Contact,`
+			`	Thank you for reaching out. I am currently out of the office and will respond to your message as soon as possible.`
+			`	For urgent matters, please contact our support team."`
+			`		Howard Vendeur`
+			`		Investor Relations Manager, Megaparsec Incorporated`
+				accept
+	on complete
+		conversation
+			`You land <ship> on <planet> and make your way to Howard's office at the Megaparsec Incorporated satellite office near the Syndicate's headquarters.`
+			`	You find yourself waiting in a sleek, sterile lobby, with the hum of distant machinery filling the air. The walls are lined with glossy advertisements for the various fast and lightweight ships, aircraft, and hovercraft that Megaparsec is known for, while a polite receptionist assures you that Howard will be with you "shortly," though you suspect that could mean anything from minutes to hours.`
+			choice
+				`	(Take a seat in the lobby.)`
+			branch makeemwait
+				has "syndicate business: mean to Howard"
+			`	Much to your surprise, the wait is brief. Within minutes, Howard emerges with a warm smile, extending a hand as he welcomes you into his office, clearly eager to discuss the next big deal.`
+				goto next
+			label makeemwait
+			`	Hours crawl by as the polished receptionist offers increasingly strained reassurances. Finally, Howard arrives, barely masking his annoyance, and motions for you to follow with a curt nod, clearly viewing this meeting as an inconvenience.`
+			label next
+			`	The inside of Howard's office is modest yet stylish, with a sleek table, a few ergonomic chairs, and a large screen cycling through market trends. The minimalist decor is designed to impress, but it's clear this is more for show than substance.`
```

Comment:
Impress who/what?
```suggestion
			`	The inside of Howard's office is modest yet stylish, with a sleek table, a few ergonomic chairs, and a large screen cycling through market trends. The minimalist decor is designed to impress anybody who comes in, but it's clear this is more for show than substance.`
```

Reply:
I was trying a riff off of the idiom "dress to impress," which doesn't have a specific who/what, but is often implied to be professional contexts (or fancy parties). Does that make this one fine or do you still it's better/more clear with the `anybody who comes in`?

Reply:
I would say it's probably more clear with my suggested change.